module-8/M8/server/client/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
remove the slash at the beginning for directories, such as "/build" and "coverage", node_modules, and so on.

This would not match directories in the current folder